pkg/manager/runtime/handler.go

```diff
@@ -47,47 +57,61 @@ func (p *Server) CreateRuntime(ctx context.Context, req *pb.CreateRuntimeRequest
 	}
 
 	// get response
-	pbRuntime, err := p.getRuntimePbWithLabel(runtimeId)
+	runtime, err := p.getRuntimePbWithLabelAndCredential(runtimeId)
```

Comment:
`getRuntimePbWithLabelAndCredential` change to `formatRuntime` like `formatRepo`